keps/sig-node/24-apparmor-ga/README.md

```diff
@@ -0,0 +1,698 @@
+      - [RuntimeDefault Profile](#runtimedefault-profile)
+- [Design Details](#design-details)
+  - [Failure and Fallback Strategy](#failure-and-fallback-strategy)
+  - [Version Skew Strategy](#version-skew-strategy)
+    - [Pod Creation](#pod-creation)
+    - [Pod Update](#pod-update)
+    - [PodTemplates](#podtemplates)
+    - [Runtime Profiles](#runtime-profiles)
+    - [Kubelet Backwards compatibility](#kubelet-backwards-compatibility)
+    - [Upgrade / Downgrade](#upgrade--downgrade)
+  - [Test Plan](#test-plan)
+  - [Graduation Criteria](#graduation-criteria)
+- [Production Readiness Review Questionnaire](#production-readiness-review-questionnaire)
+  - [Feature enablement and rollback](#feature-enablement-and-rollback)
+  - [Rollout, Upgrade and Rollback Planning](#rollout-upgrade-and-rollback-planning)
+  - [Monitoring requirements](#monitoring-requirements)
+  - [Dependencies](#dependencies)
+  - [Scalability](#scalability)
+  - [Troubleshooting](#troubleshooting)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+Items marked with (R) are required _prior to targeting to a milestone / release_.
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in
+      [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and
+      SIG Testing input
+- [ ] (R) Graduation criteria is in place
+- [ ] (R) Production readiness review completed
+- [ ] Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for
+      publication to [kubernetes.io]
+- [ ] Supporting documentation e.g., additional design documents, links to
+      mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+This is a proposal to upgrade the AppArmor annotation on pods to a dedicated
+field, and graduate the feature to GA. This proposal aims to do the _bare
+minimum_ to clean up the feature, without blocking future enhancements.
+
+## Motivation
+
+AppArmor support has been added with Kubernetes v1.4 and is already in beta.
+Profiles have to be available on each node whereas the container runtime ensures
+that the profile is loaded when specified at pod level. Profiles can be
+specified per-container via the pod’s metadata annotation:
```

Comment:
We should describe the motivation that prompted adding AppArmor, not the motivation to graduate the feature. It's OK to borrow text from the discussion in https://github.com/kubernetes/enhancements/issues/24.

Reply:
Added more color on the motivation, as well as changed wording to adding AA support, rather than bringing to GA in [9b5bcff](https://github.com/kubernetes/enhancements/pull/3298/commits/9b5bcff6446d7ac402e31d5a986104912c71a999)